hw/xbox/dsp/dsp_cpu.c

```diff
@@ -159,6 +159,9 @@ static bool match_MMMRRR(uint32_t op)
     return true;
 }
 
+static OpcodeEntry* search_opcode(uint32_t op);
+OpcodeEntry *OpcodeEntryArray[0xFFFFF+1]; //for the lower 20bits of the opcodes
```

Comment:
The interesting comment would be: *why* only the lower 20 bits?

Reply:
alright 
hope this is better: `/*nonparallel opcodes only occupy the lower 20bits*/
	`